src/layout/MasterLayout.tsx

```diff
@@ -0,0 +1,16 @@
+
+import { Outlet } from 'react-router-dom';
+import '../assets/styles/App.css'
+
+const MasterLayout = () => {
```

Comment:
please move Sidebar and menu components to this menu, any layout component should be in this component (in one place)